.github/workflows/deploy-sandbox.yml

```diff
@@ -122,7 +126,8 @@ jobs:
           --layer-name "Datadog-Extension" \
           --layer-suffix "${{ inputs.layerSuffix }}" \
           --region ${{ github.event.inputs.region }} \
-          --key ${{ secrets.GH_ACTION_PUBLISHER_AEM_KEY }}
+          --key ${{ secrets.GH_ACTION_PUBLISHER_AEM_KEY }} \
+          --description "${{ inputs.description }}"
```

Comment:
did you the GitHub action when description is empty?

Reply:
I don't know how to test this actually! is there a good way?

Reply:
you can write a integration test or manually trigger the workflow locally using https://cli.github.com/manual/gh_workflow_run